polymer_elements_config.yaml

```diff
@@ -176,9 +176,14 @@ files_to_generate:
   - google-chart/google-chart.html:
       omit_imports:
         - promise-polyfill-lite.html
-  - google-chart/charts-loader.html:
```

Comment:
Seams that `chart-loader.html` is now imported by `google-chart-loader.html`. Both are not elements neither intended to be used directly so I thought it was better hide it. It will be used in the js side thought